user_guides/connector/src/main/jbake/templates/footer.ftl

```diff
@@ -37,7 +37,8 @@
 
 <span id="copyright">
 		<img src="img/eclipse_foundation_logo_tiny.png" height="20px" alt="Eclipse Foundation Logo" align="top"/>&nbsp;			
-		<span >Copyright&nbsp;&copy;&nbsp;2018,&nbsp;2020&nbsp;Oracle&nbsp;and/or&nbsp;its&nbsp;affiliates.&nbsp;All&nbsp;rights&nbsp;reserved.</span>
+		<span >Copyright&nbsp;&copy;&nbsp;2020,&nbsp;Oracle&nbsp;and/or&nbsp;its&nbsp;affiliates.&nbsp;All&nbsp;
```

Comment:
I believe this should have been "Copyright&nbsp;&copy;&nbsp;2018,&nbsp;2020&nbsp;". @alwin-joseph , please correct me If this is not the case.

Reply:
I think you are right @RohitKumarJain !
I saw the same line at [jax ws footer](https://github.com/eclipse-ee4j/jakartaee-tck/blob/accb75d6d8c4fa2ac9f1be096f4db86e156a9e31/user_guides/jaxws/src/main/jbake/templates/footer.ftl#L40)
Thanks for your review!